packages/page-accounts/src/Accounts/index.tsx

```diff
@@ -71,23 +72,42 @@ function Overview ({ className = '', onStatusChange }: Props): React.ReactElemen
   });
   const isLoading = useLoadingDelay();
 
+  const _sort = (header: SortCategory) => {
+    setSortBy(({ sortBy, sortFromMax }) =>
+      ({ sortBy: header,
+        sortFromMax: sortBy === header ? !sortFromMax : true }));
+  };
+
+  const sortableHeader = (header: SortCategory, clas?: string) => [
```

Comment:
class?

Reply:
htmlClass?

Reply:
I did not wanted to use `class` because it could conflict with the keyword. But `htmlClass` is the best here